closed/src/java.base/share/classes/sun/security/ec/NativeECKeyPairGenerator.java

```diff
@@ -170,13 +170,10 @@ public KeyPair generateKeyPair() {
         long nativePointer = NativeECUtil.encodeGroup(this.params);
 
         if (nativePointer == -1) {
-            absent = NativeECUtil.putCurveIfAbsent(this.curve, Boolean.FALSE);
```

Comment:
Please remove the (now unused) local variable `absent`.

Reply:
Done.